src/routes/index.ts

```diff
@@ -0,0 +1,16 @@
+import { Request, Response, Router } from 'express';
+
+import home from './home';
+
+const router = Router();
+
+// add your router function here
+router.use('/', home);
+router.use('*', (request: Request, response: Response) => {
+  return response.status(404).json({
+    status: 'error',
+    message: 'Oops, page not found',
+  });
+});
+
+export default router;
```

Comment:
We'll need a linting rule for end-of-line. Are we using AirBnB? I think it has it.

Reply:
Done